### PR TITLE
[HOLD] Show bubble when engagement is restored

### PR DESCRIPTION
**Pull request by igorkravchenko**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-2648

**What was solved?**
Show bubble when engagement is restored to align behaviour with Android Widgets.
A few points to pay attention to:
- After configuration if there's ongoing engagement, bubble view will show up because engagement gets restored
In order to align the behaviour with Android, if there's existing engagement we resume it when `Glia.sharedInstance.startEngagement(engagementKind:in:features:sceneProvider:)` is called.
- Now we also provide `features` parameter with `Glia.sharedInstance.configure` method.

Also #1011  was added into this PR since it makes changes to configuration as well.

**Release notes:**

 - [ ] Feature
 - [ ] Ignore
 - [x] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

- [ ] Is the feature sufficiently tested? All tests fixed? Necessary unit, acceptance, snapshots added? Check that at least new public classes & methods are covered with unit tests
 - [ ] Did you add logging beneficial for troubleshooting of customer issues?
 - [ ] **Did you add new logging?** We would like the logging between platforms to be similar. Refer to [**Logging from iOS SDKs** → **Things to consider for newly added logs**](https://glia.atlassian.net/wiki/spaces/ENG/pages/3589734507/Logging+from+iOS+SDKs#Things-to-consider-for-newly-added-logs) in Confluence for more information.

**Screenshots:**

### CODE REVIEW

**Review by EgorovEI — APPROVED** [review]

**Comment by igorkravchenko** [comment]
!squash

File: GliaWidgets/Public/Glia/Glia+RestoreEngagement.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        // be appeared again.
+        createdInteractor.skipLiveObservationConfirmations = true // TODO: snack bar has to be shown, but not dialog
+
+        // Apply company name to theme and get the modified theme.
+        let modifiedTheme = applyCompanyName(using: configuration, theme: theme)
+
+        let viewFactory = ViewFactory(
+            with: modifiedTheme,
+            messageRenderer: messageRenderer,
+            environment: .init(
```
Comment (rasmustautsglia) [inline GliaWidgets/Public/Glia/Glia+RestoreEngagement.swift:25]: Can environment be initialized in a static method within `viewFactory` environment's extension?

Comment (igorkravchenko) [reply GliaWidgets/Public/Glia/Glia+RestoreEngagement.swift]: Updated to use it 07047ccf370e05a605a583b15d64237dd3d6e307.

Comment (igorkravchenko) [reply GliaWidgets/Public/Glia/Glia+RestoreEngagement.swift:25]: Addressed this in #996 via a02dcc8b22d9355ca40af7e846e873106bf90f18.

**Review by rasmustautsglia — APPROVED** [review]

**Pull request merged**